src/lektorium/repo/local/repo.py

```diff
@@ -56,6 +93,16 @@ def parked_sessions(self):
                 if session.parked:
                     yield session
 
+    @property
+    def releasing(self):
+        for site_id, site in self.config.items():
+            for merge_request_data in self.storage.get_merge_requests(site_id):
+                if merge_request_data['source_branch'].startswith('session-'):
```

Comment:
May be safer to convert to lower case before `startswith` ( `merge_request_data['source_branch'].lower()`)

Reply:
Why do you think so?

Reply:
It it is the only possible source of naming, then ok